<?xml version = "1.0"?>

<launch>
    <!-- run pointcloud_to_laserscan node -->
    <node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan">

        <remap from="cloud_in" to="/pcl_cloud"/>
        <rosparam>
            target_frame: base_footprint
            transform_tolerance: 0.01
            min_height: 0.0
            max_height: 5.0

            angle_min: -3.14 # -M_PI
            angle_max: 3.14 # M_PI
            angle_increment: 0.0087 # M_PI/360.0
            scan_time: 0.3333
            range_min: 0.0
            range_max: 10.0
            use_inf: true

        </rosparam>
    </node>
</launch>